packages/components/src/popover/index.tsx

```diff
@@ -252,6 +253,17 @@ const UnforwardedPopover = (
 		} );
 	}
 
+	let computedVariant = variant;
+	if ( isAlternate !== undefined ) {
+		deprecated( '`isAlternate` prop in wp.components.Popover', {
+			since: '6.2',
+			version: '6.4',
+			alternative: '`variant` prop with the `alternate` value',
+		} );
+
+		computedVariant = isAlternate ? 'alternate' : computedVariant;
```

Comment:
```suggestion
		const computedVariant = isAlternate ? 'alternate' : variant;
```

Could we just declare a const here?

Reply:
`const` is block scoped, so unfortunately not.

Reply:
Sorry, I meant declare it as const where the `let` is right now.

Reply:
Oh yep, true. I've updated it to do that 👍